ReactWindows/ReactNative.Shared/DevSupport/DevInternalSettings.cs

```diff
@@ -130,14 +135,25 @@ private T GetSetting<T>(string key, T defaultValue)
                     return (T)data;
                 }
             }
+#else
+            var value = ReactNative.Net46.Properties.Settings.Default[key];
+            if (value is T)
+            {
+                return (T)value;
+            }
+#endif
 
             return defaultValue;
         }
 
         private void SetSetting<T>(string key, T value)
         {
+#if WINDOWS_UWP
             var values = ApplicationData.Current.LocalSettings.Values;
             values[key] = value;
+#else
+            ReactNative.Net46.Properties.Settings.Default[key] = value;
+#endif
```

Comment:
I'm assuming you've tested this in some way?  Does the generated Settings class actually support arbitrary keys without having added anything to the designer?